src/core.js

```diff
@@ -421,7 +421,7 @@
             try {
                var loginUrl = createLoginUrl(params);
                connect.getLog().warn("ACK_TIMEOUT occurred, attempting to pop the login page if not already open.");
-               connect.core.getPopupManager().open(loginUrl, connect.MasterTopics.LOGIN_POPUP);
+               connect.core.loginWinow = connect.core.getPopupManager().open(loginUrl, connect.MasterTopics.LOGIN_POPUP);
```

Comment:
There is a little typo in "loginWinow" => "loginWindow"

Reply:
Fixed, thanks for pointing out